fitvd/fofs.py

```diff
@@ -126,17 +145,28 @@ def _init_bounds_by_radius(self):
 
         r += self.conf['padding_arcsec']
 
-        # factor of 2 because this should be a diameter as it is used later
-        diameter = r*2
-        self.sze = diameter
```

Comment:
You are not setting sze any more. May not matter

Reply:
that was not used anywhere in the code; I don't recall how it was used in your original